Jenkinsfile

```diff
@@ -0,0 +1,260 @@
+#!groovy
+
+node {
+    def TIDB_TEST_BRANCH = "master"
+    def TIDB_BRANCH = "master"
```

Comment:
Unfortunately it's not supported by Jenkinsfile.
But only long time release branch (master, rc2.2) needs a Jenkinfile.
Before merging a long time branch into another one, you can submit a pr to remove it.